build/lib/test/test.js

```diff
@@ -518,6 +602,11 @@ async function handleBuild(prc, target, snapshotDir, snapshotPromises) {
 	});
 }
 
+/**
+ *
+ * @param {string} testResults
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 607;
-->

- :warning: build/lib/test/test.js line 607 – Missing JSDoc parameter description for 'testResults'. (valid-jsdoc)